pkg/engine/diff.go

```diff
@@ -444,26 +444,40 @@ func shortHash(hash string) string {
 }
 
 func printOldNewDiffs(
-	b *bytes.Buffer, olds resource.PropertyMap, news resource.PropertyMap,
+	b *bytes.Buffer, olds resource.PropertyMap, news resource.PropertyMap, include []resource.PropertyKey,
 	planning bool, indent int, op deploy.StepOp, summary bool, debug bool) {
 
 	// Get the full diff structure between the two, and print it (recursively).
 	if diff := olds.Diff(news); diff != nil {
-		printObjectDiff(b, *diff, planning, indent, summary, debug)
+		printObjectDiff(b, *diff, include, planning, indent, summary, debug)
 	} else {
 		// If there's no diff, report the op as Same - there's no diff to render
 		// so it should be rendered as if nothing changed.
 		printObject(b, news, planning, indent, deploy.OpSame, true, debug)
 	}
 }
 
-func printObjectDiff(b *bytes.Buffer, diff resource.ObjectDiff,
+func printObjectDiff(b *bytes.Buffer, diff resource.ObjectDiff, include []resource.PropertyKey,
 	planning bool, indent int, summary bool, debug bool) {
 
 	contract.Assert(indent > 0)
 
-	// Compute the maximum with of property keys so we can justify everything.
+	// Compute the maximum with of property keys so we can justify everything. If an include set was given, filter out
```

Comment:
SUPER NIT: Width instead of with.

Reply:
I will fix this broken window for you :)